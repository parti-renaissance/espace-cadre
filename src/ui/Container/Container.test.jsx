import { render } from '@testing-library/react'
import UIContainer from './Container'

vi.mock('@mui/material', () => ({
  Grid: ({ children, sx }) => <div data-sx={sx}>{children}</div>,
}))
vi.mock('@mui/system', () => ({
  styled: c => () => c,
}))

describe('Container', () => {
  it('displays grid', () => {
    const { container } = render(<UIContainer rootProps={{ sx: 'foo' }}>Foo</UIContainer>)

    expect(container).toMatchSnapshot()
  })
})
