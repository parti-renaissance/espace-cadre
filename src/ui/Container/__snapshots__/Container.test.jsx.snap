// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Container > displays grid 1`] = `
<div>
  <div
    data-sx="foo"
  >
    Foo
  </div>
</div>
`;

exports[`Container displays grid 1`] = `
<div>
  <div
    data-sx="foo"
  >
    Foo
  </div>
</div>
`;
